lib/ae_mdw/db/mutations/derive_aex9_presence_mutation.ex

```diff
@@ -45,6 +49,7 @@ defmodule AeMdw.Db.DeriveAex9PresenceMutation do
       m_balance =
         Model.aex9_balance(
           index: {contract_pk, account_pk},
+          block_index: block_index,
```

Comment:
How does this affect already existing aex9_balances? Shouldn't we have a migration for them as well?

Reply:
It requires a full sync.